src/cli/encrypt.go

```diff
@@ -0,0 +1,27 @@
+package cli
+
+import "github.com/spf13/cobra"
+
+const encryptShortDescription = "Find unencrypted values in configurations and encrypt them"
+const encryptLongDescription = `Command "encrypt"
+
+This command will look for unencrypted values in all local configurations and updates them by encrypting them.
```

Comment:
Nikto rad necita dlhe vety, ... hlavne v EN to pisu vacsinou co najkratsie, nemoze to byt napr.?
`This command searches for unencrypted values in local configurations and encrypts them.`

Reply:
... alebo `in all local`